static/app/actionCreators/group.tsx

```diff
@@ -214,7 +214,7 @@ export function updateNote(
 type ParamsType = {
   environment?: string | string[] | null;
   itemIds?: string[];
-  project?: number[] | null;
+  project?: number[] | string[] | null;
```

Comment:
`project` is an array of project IDs, which can be a number or string (it ends up in the query params so it doesn't really matter, I changed the type because the group project ID is typed as a string).

Wish it was labeled a bit better but it's used in many places so won't change it now